examples/wave-eager.py

```diff
@@ -24,35 +24,37 @@
 import numpy.linalg as la  # noqa
 import pyopencl as cl
 import pyopencl.array as cla  # noqa
-import pyopencl.clmath as clmath
 from pytools.obj_array import join_fields
 from grudge.eager import EagerDGDiscretization
 from grudge.shortcuts import make_visualizer
 from mirgecom.wave import wave_operator
 from mirgecom.integrators import rk4_step
+from meshmode.array_context import PyOpenCLArrayContext
+from meshmode.dof_array import thaw
 
 
-def bump(discr, queue, t=0):
+def bump(discr, actx, t=0):
```

Comment:
I've stuck to the convention `actx, discr` in the calling sequence in grudge otherwise.

Reply:
Fixed, thanks.